woocommerce/woocommerce-gutenberg-products-block
# Add Product Collection Grid pattern

## What

Add Product Collection Grid pattern

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/9368

## Why

This is one of the patterns from a set of patterns we'd like to provide for Product Collection

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

0. At this moment Product Collection is still experimental feature (PR to make it Core feature: https://github.com/woocommerce/woocommerce-blocks/pull/10524), hence the pattern is not included in the inserter. In order to test the pattern please remove this line: https://github.com/woocommerce/woocommerce-blocks/pull/10660/files#diff-d26dedf0a7efa88eae30cbe732192f2769376839845875e6e57505d33f19653cR6.
1. Go to Editor
3. Insert "Product Collection Grid" pattern
4. Save and go to frontend
5. Verify it looks like in the screenshots below

Known limitations:
- Product Price cannot be styled according to designs. This work is being covered in https://github.com/woocommerce/woocommerce-blocks/pull/9398


## Screenshots or screencast



Reference | Editor | Frontend |
| ------ | ----- | ---|
| ![image](https://github.com/woocommerce/woocommerce-blocks/assets/20098064/6264f504-379e-4441-84c7-08bd4b647e2b) |    <img width="1431" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/20098064/e1aa1aea-5960-439d-a6d5-0437825c0879">    |    <img width="1312" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/20098064/dd3bd4c1-1eeb-42c5-a94e-c0aa33f15441">   |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Product Collection: new pattern "Product Collection Grid"
